staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go

```diff
@@ -58,13 +60,19 @@ func NamedCheck(name string, check func(r *http.Request) error) HealthzChecker {
 	return &healthzCheck{name, check}
 }
 
-// InstallHandler registers a handler for health checking on the path "/healthz" to mux.
+// InstallHandler registers handlers for health checking on the path
+// "/healthz" to mux. *All handlers* for mux must be specified in
+// exactly one call to InstallHandler. Calling InstallHandler more
+// than once for the same mux will result in a panic.
 func InstallHandler(mux mux, checks ...HealthzChecker) {
 	if len(checks) == 0 {
+		glog.Info("No default health checks specified. Installing the ping handler.")
 		checks = []HealthzChecker{PingHealthz}
 	}
+	glog.Infof("Installing %v healthz checkers", len(checks))
```

Comment:
@sttts my point being, I can't just throw the variadic `checks` slice into the logger and call it a day. I have to loop over each interface and call `Name()` on it, accumulating the result into string before finally logging it outside the loop.

I didn't originally have the log line in the loop, but I do kind of like it as any panic should be more exactly associated with a log line of the checker that caused it.